crates/sncast/src/response/structs.rs

```diff
@@ -66,6 +66,7 @@ impl CommandResponse for AccountCreateResponse {}
 #[derive(Serialize)]
 pub struct AccountImportResponse {
     pub add_profile: String,
+    pub account_name: Option<String>,
```

Comment:
This is added so we can show the generated account name on the terminal when `--name` flag is skipped.